test/Dialect/FIRRTL/SFCTests/data-taps-xmr.fir

```diff
@@ -83,13 +83,13 @@ circuit Top : %[[
 
     inst foo of Foo
 
-    wire tap : UInt<2>
+    wire tap : UInt<1>
     tap is invalid
 
-    wire tap2 : UInt<4>
+    wire tap2 : UInt<1>
     tap2 is invalid
 
-    wire tap3 : UInt<5>
+    wire tap3 : UInt<1>
```

Comment:
Any reason for updating the widths ? Seems harmless though.

Reply:
This test was toggled recently, I'll go back since AFAIK doesn't matter functionally and helps debugging.